tests/test_sql_refsols/simple_range_1_ansi.sql

```diff
@@ -0,0 +1,13 @@
+SELECT
```

Comment:
Ignore generated ANSI and SQLITE files here. They will be removed before merging.